scripts/version

```diff
@@ -13,8 +14,12 @@ else
     VERSION="0.0.0-${COMMIT}${DIRTY}"
 fi
 
-if [ -z "$ARCH" ]; then
+if [ -z "${ARCH:-}" ] ; then
     ARCH=$(go env GOHOSTARCH)
+else
+    case "$ARCH" in
+     linux/*) ARCH="${ARCH#*/}" ;;
+    esac
```

Comment:
That's odd, where have you seen this being needed?

On my machine, `go env GOHOSTARCH` only reports `amd64`, not `linux/amd64`.

Reply:
This was temporary, not needed anymore.